packages/nouns-subgraph/README.md

```diff
@@ -37,3 +37,36 @@ yarn codegen
 ```sh
 yarn deploy:[network] # Supports rinkeby and mainnet
 ```
+
+## Running a local deployment
```

Comment:
Worth mentioning that the local chain must be started by running `yarn task:run-local` from the contracts directory.

Reply:
fixed